custom_components/miele/sensor.py

```diff
@@ -174,14 +174,29 @@ class MieleSensorDefinition:
     MieleSensorDefinition(
         types=[
             WASHING_MACHINE,
+            WASHER_DRYER,
+        ],
+        description=MieleSensorDescription(
+            key="targetTemperature",
+            data_tag="state|targetTemperature|0|value_raw",
+            device_class=SensorDeviceClass.TEMPERATURE,
+            name="Target temperature",
+            native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+            state_class=SensorStateClass.MEASUREMENT,
```

Comment:
```suggestion

```
This sensor is not a measurement according to the spec in https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes. It should be removed from other targetTemperature sensors also. I can do it in another PR.